host/distro/distro.go

```diff
@@ -39,11 +39,11 @@ func IsArmbian() bool {
 func IsDebian() bool {
 	if isLinux {
 		// http://0pointer.de/public/systemd-man/os-release.html#ID_LIKE=
-		id, _ := OSRelease()["ID"]
+		id := OSRelease()["ID"]
 		if id == "debian" {
 			return true
 		}
-		idLike, _ := OSRelease()["ID_LIKE"]
+		idLike := OSRelease()["ID_LIKE"]
 		for _, part := range strings.Split(idLike, " ") {
```

Comment:
`for _, part := range strings.Split(OSRelease()["ID_LIKE"], " ") {`
and same below.